public/components/datasources/components/manage/inactive_data_connection.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { EuiButton, EuiCallOut } from '@elastic/eui';
+import React from 'react';
+import {
+  DATACONNECTIONS_BASE,
+  DATACONNECTIONS_UPDATE_STATUS,
+  EDIT,
+} from '../../../../../common/constants/shared';
+import { coreRefs } from '../../../../framework/core_refs';
+import { useToast } from '../../../common/toast';
+import { DatasourceDetails } from './data_connection';
+
+interface InactiveDataConnectionCalloutProps {
+  datasourceDetails: DatasourceDetails;
+  fetchSelectedDatasource: () => void;
+}
+
+export const InactiveDataConnectionCallout = ({
+  datasourceDetails,
+  fetchSelectedDatasource,
+}: InactiveDataConnectionCalloutProps) => {
+  const { setToast } = useToast();
+  const { http } = coreRefs;
+  const enableDataSource = () => {
+    http!
+      .post(`${DATACONNECTIONS_BASE}${EDIT}${DATACONNECTIONS_UPDATE_STATUS}`, {
+        body: JSON.stringify({ name: datasourceDetails.name, status: 'active' }),
+      })
+      .then(() => {
+        setToast(`Data connection ${datasourceDetails.name} enabled successfully`);
+        fetchSelectedDatasource();
+      })
+      .catch((err) => {
+        console.error(err);
+        setToast(`Data connection ${datasourceDetails.name} could not be enabled.`, 'danger');
+      });
+  };
+
+  return (
+    <EuiCallOut title="This data source connection is inactive" color="warning" iconType="help">
```

Comment:
please add i18n here and below user facing text

Reply:
ack. I was talking to Paul about it we should have a meta issue for all of dashboards-observability to follow i18n. 
https://github.com/opensearch-project/dashboards-observability/issues/1539